package.json

```diff
@@ -14,14 +14,19 @@
     "url": "git@github.com:botify-labs/react-classmap.git"
   },
   "license": "MIT",
+  "devEngines": {
+    "node": "4.x"
+  },
   "dependencies": {
-    "react": "^0.13.3"
+    "react": "^0.14.0",
```

Comment:
`"^0.14.0"` will only allow patch updates. However, since we're requiring private modules, we might want to fix the version to `"0.14.0"`.